### PR TITLE
Fix git centos7 require curl devel to support https

### PR DESCRIPTION
**Pull request by clementperon**

### CODE REVIEW

**Comment by seladb** [comment]
@clementperon can you please update from `master`? It already contains testing before pushing to docker-hub

**Comment by clementperon** [comment]
@seladb done

**Comment by seladb** [comment]
@clementperon I forgot to mention - currently the CI builds `ubuntu2004`, you should change it to `centos7` to test the change:
https://github.com/clementperon/PcapPlusPlus-DockerImages/blob/7ac82535c3685615809daddf6ecd811617020d42/.github/workflows/docker-image.yml#L14

File: .github/workflows/docker-image.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ concurrency:
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: centos7
```
Comment (seladb) [inline .github/workflows/docker-image.yml:10]: you accidentally changed `runs-on` instead of `image`...

**Pull request merged**